packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -66,6 +67,8 @@ class CreateCommand extends Command {
       return 2;
     }
 
+    await Cache.instance.updateAll();
```

Comment:
No.  Create still depends on pub.dartlang.org.  We'll need to internalize our Dart package dependencies before it will be possible to run create offline.